app/assets/javascripts/app/views/twitter/account_edit.jst.eco

```diff
@@ -31,4 +31,8 @@
   <p class="description"><%- @T('Choose which group %s will get added to.', 'direct messages') %></p>
   <div class="js-directMessagesGroup"></div>
 
+  <h3>Retweets</h3>
+  <p class="description"><%- @T('Choose if %s should also be converted to tickets.', 'retweets') %></p>
```

Comment:
Shouldn't be 'retweets' in the translation string to it could get translated in e.g. mandarin?